### ISSUE TITLE
Can connect to inspector even with DebugMode(false)

### ISSUE THREAD

**Issue by DevJac** [sciter-sdk/rust-sciter#54]
I run the following before creating my first Sciter window:
```
    sciter::set_options(sciter::RuntimeOptions::DebugMode(false)).unwrap();
    sciter::set_options(sciter::RuntimeOptions::ScriptFeatures(
        sciter::SCRIPT_RUNTIME_FEATURES::ALLOW_SOCKET_IO as u8,
    ))
    .unwrap();
```
I am still able to connect to the inspector even though I set `DebugMode(false)`.

I can provide a small example if desired.

**Comment by pravic** [comment]
Yes, the debug mode is always set. I need to move that to options.

**Comment by pravic** [comment]
Interestingly , that this issue contradicts to #52.

**Comment by pravic** [comment]
Was fixed in 9cde705 by removing the implicit debug mode during the frame creation.

Also, the `ALLOW_SOCKET_IO ` isn't necessary anymore (it was only needed between 4.2.7.0 and 4.2.5.0).
